go.mod

```diff
@@ -1,26 +1,20 @@
 module github.com/snapcore/snapd
 
-// TODO: update to 1.18, doing this naively breaks the builds because it
-// seems this enables automatic vendoring, see
-// https://go.dev/ref/mod#go-mod-file-go
-go 1.13
+go 1.18
 
 // maze.io/x/crypto/afis imported by github.com/snapcore/secboot/tpm2
 replace maze.io/x/crypto => github.com/snapcore/maze.io-x-crypto v0.0.0-20190131090603-9b94c9afe066
 
 require (
-	github.com/alexkohler/nakedret v1.0.1 // indirect
```

Comment:
Is nakedret no longer required when using go 1.18, or does go 1.18 break nakedret?

Reply:
I actually am not sure, I think it just removes all "indirect" dependencies from "require"